infra/core/host/function.bicep

```diff
@@ -0,0 +1,127 @@
+metadata description = 'Creates an Azure Function in an existing Azure App Service plan.'
```

Comment:
Did you copy this from latest?  The latest is functions.bicep, not function.bicep, so I'm a little concerned that core wasn't copied from the latest.  Can you double check all files are latest?

<img width="166" alt="image" src="https://github.com/jongio/chat-with-your-data-solution-accelerator/assets/2163001/0be6aef2-4b71-4324-bc18-cad0f94e9f35">

https://github.com/Azure/azure-dev/tree/main/templates/common/infra/bicep/core/host